guides/common/modules/proc_installing-and-configuring-the-puppet-agent.adoc

```diff
@@ -12,7 +12,7 @@ endif::[]
 .Prerequisites
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite[]
-* The {ProjectName} Client Puppet 7.0 repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
+* The {ProjectName} Client {ProjectVersion} repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
```

Comment:
With 6.11 there is a version-less client repository being introduced. Therefore this should read:

```suggestion
* The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
```